pkg/pillar/zboot/zboot.go

```diff
@@ -256,11 +256,20 @@ func GetPartitionSizeInBytes(partName string) uint64 {
 	validatePartitionName(partName)
 	_, ok := partDev[partName]
 	if ok {
-		partsize, err := execWithRetry(nil, "zboot", "partdevsize", partName)
+		ret, err := execWithRetry(nil, "zboot", "partdevsize", partName)
 		if err != nil {
 			logrus.Fatalf("zboot partdevsize %s: err %v\n", partName, err)
```

Comment:
No need for \n at the end of the log strings; log is in json.

Reply:
ok